docs/concepts/workloads/controllers/daemonset.md

```diff
@@ -75,7 +75,7 @@ a node for testing.
 If you specify a `.spec.template.spec.nodeSelector`, then the DaemonSet controller will
 create pods on nodes which match that [node
 selector](/docs/user-guide/node-selection/). Likewise if you specify a `.spec.template.spec.affinity` 
-then DaemonSet controller will create pods on nodes which match that [node affinity](../../user-guide/node-selection/index.md).
+then DaemonSet controller will create pods on nodes which match that [node affinity](/docs/user-guide/node-selection/).
```

Comment:
That page was moved, so you should probably use a direct link to the current page:

`/docs/concepts/configuration/assign-pod-node/`